src/libstd/sys_common/backtrace.rs

```diff
@@ -34,6 +33,7 @@ pub fn lock() -> impl Drop {
 }
 
 /// Prints the current backtrace.
+#[cfg(feature = "backtrace")]
```

Comment:
Instead of `#[cfg]` here, could this also update [this section of code](https://github.com/rust-lang/rust/blob/b6269f27d99d7da9e95f0b3fdc53193dc8c42fbe/src/libstd/panicking.rs#L160-L206) to use `if cfg!(feature = "backtrace")` instead of `#[cfg]`?